### PR TITLE
Add a new way to display size (smaller)

### PR DESCRIPTION
**Pull request by meain**
What do you  think of an option like this?

### CODE REVIEW

File: src/flags.rs [hunk 1]
```diff
@@ -117,6 +121,22 @@ impl Default for Flags {
...
+#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+pub enum SizeFlag {
+    Default,
+    Small,
+}
+
+impl<'a> From<&'a str> for SizeFlag {
+    fn from(size: &'a str) -> Self {
+        match size {
+            "size" => SizeFlag::Default,
```
Comment (Peltoche) [inline src/flags.rs]: What do you think of replacing "size" by something more explicit like "classic" or even "default"?

**Comment by Peltoche** [comment]
Good idea! This can be a good start for proposing a "compressed mode" used for some layout like the `exa -lG`.

File: src/flags.rs (hunk 1 above)
Comment (meain) [reply src/flags.rs]: I think we will use `default` and maybe we could display raw number of bytes for `classic`?

**Comment by meain** [comment]
@Peltoche I know this got merged but I feel like `short` might be a better flag than `small`. What do you think? Should we change it to short?

**Comment by Peltoche** [comment]
> @Peltoche I know this got merged but I feel like `short` might be a better flag than `small`. What do you think? Should we change it to short?

It's true that "short" seems semantically more correct to me. As the feature have not been released yet, we can change the name without problems so let's go for it.

**Pull request merged**